pages/landing/enterprise-eks/index.html

```diff
@@ -219,6 +223,182 @@
+        eks: "no"
+        gruntwork: "yes"
+      - name: "App monitoring"
+        description: "App metrics, dashboards, and log aggregation are available out-of-the-box."
+        kops: "no"
+        rancher: "yes"
+        eks: "no"
+        gruntwork: "yes"
+  - title: Security
+    features:
+      - name: "Server hardening"
+        description: "All servers (worker nodes) are implement security best practices out-of-the-box, including OS hardening, SSH key management, fail2ban, NTP, and more."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
+      - name: "Sealed secrets"
+        description: "Manage Kubernetes secrets as code by sealing them in your repositories with AWS KMS encryption."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
+      - name: "End-to-end encryption"
+        description: "All data is encrypted on disk and in-transit (e.g., service-to-service communication), out-of-the-box."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
+      - name: "Service communication ACLs"
+        description: "All communication between services is tightly controlled via service mesh ACLs, out-of-the-box."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
+      - name: "Network security policies"
+        description: "Best-practice network security policies are installed, out-of-the-box."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
+      - name: "Pod security policies"
+        description: "Best-practice pod security policies are installed, out-of-the-box."
+        kops: "no"
+        rancher: "yes"
+        eks: "no"
+        gruntwork: "yes"
+      - name: "OPA policies"
+        description: "Support for enforcing organization governance and compliance through Open Policy Agent is available out-of-the-box."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
+  - title: Compliance
+    features:
+      - name: "AWS Well Architected"
+        description: "The Kubernetes cluster follows AWS guidelines and latest industry best practices."
+        kops: "yes"
+        rancher: "yes"
+        eks: "yes"
+        gruntwork: "yes"
+      - name: "CIS AWS Foundations Benchmark"
+        description: "The Kubernetes cluster is compliant with the CIS AWS Foundations Benchmark, out-of-the-box. The CIS AWS Foundations Benchmark is an objective, consensus-driven guideline for establishing secure infrastructure on AWS."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
+      - name: "CIS EKS Benchmark"
+        description: "The Kubernetes cluster is compliant the CIS EKS Benchmark, out-of-the-box. The CIS EKS Benchmark is an objective, consensus-driven guideline for establishing a secure EKS cluster."
+        kops: "no"
+        rancher: "yes"
+        eks: "no"
```

Comment:
Hm, I'm not a 100% sure if it's out-of-the-box, but I am confused if this is EKS CIS benchmark, isn't EKS also meeting that straightaway? I can't find on AWS's website if they say that explicitly though.

Reply:
No, it doesn't support it out-of-the-box. You have to do a ton of work to meet the benchmark.